src/mudlet-lua/lua/Other.lua

```diff
@@ -636,3 +636,22 @@ function shms(seconds, bool)
 		return hh, mm, ss
 	end
 end
+
+-- returns true if your Mudlet is older than the given version
+-- for example, it'll return true if you're on 2.1 and you do mudletOlderThan(3,1)
+-- it'll return false if you're on 4.0 and you do mudletOlderThan(4,0,0)
+function mudletOlderThan(inputmajor, inputminor, inputpatch)
+  local mudletmajor, mudletminor, mudletpatch = getMudletVersion("table")
```

Comment:
😊  It is embarrassing but I really did cock-up the optional arguments for this command - no arguments produces a lua associative table containing four string-keys "major", "minor", "patch" and "build" whereas the return for a supplied argument of "table" is a multi-return set of four return values with those items - "build" being a __nil__ for "release" versions in either case (so it disappears anyway)...!